### PR TITLE
Update packages page

### PR DESCRIPTION
**Pull request by helenmasters**
Created infrastructure card, fixed typos, removed SwiftyJSON.

### CODE REVIEW

File: packages.html [hunk 1]
```diff
@@ -69,29 +69,11 @@ <h2><a name="all" style="padding-top: 10px"></a>ALL REPOS</h2>
...
-          <h3><a href="https://github.com/IBM-Swift/BlueCryptor">BlueCryptor</a></h3>
-          <h4>Cross-platform cryptography library.</h4>
-
-          <h3><a href="https://github.com/IBM-Swift/BlueSignals">BlueSignals</a></h3>
-          <h4>Generic cross-platform signal handler.</h4>
-
-          <h3><a href="https://github.com/IBM-Swift/BlueRSA">BlueRSA</a></h3>
-          <h4>Cross-platform RSA wrapper library.</h4>
+          <h3><a href="https://github.com/IBM-Swift/KituraKit">KituraKit</a></h3><a href="https://ibm-swift.github.io/KituraKit/"><h5>API</h5></a>
+          <h4>Client side library for using codable routes with Kitura.</h4>
```
Comment (ianpartridge) [inline packages.html]: Codable not codable

File: packages.html [hunk 2]
```diff
@@ -221,33 +195,61 @@ <h4>Sample application demonstrating Kitura and IBM Swift features.</h4>
...
           <h3><a href="https://github.com/IBM-Swift/swift-html-entities">swift-html-entities</a></h3>
           <h4>Pure Swift HTML encode and decode utility tool.</h4>
 
           <h3><a href="https://github.com/IBM-Swift/Swift-JWT">Swift-JWT</a></h3>
-          <h4>An implementation of JSON Web Tokens in pure Swift.</h4>
+          <h4>An implementation of JSON Web Tokens in Swift.</h4>
 
-          <h3><a href="https://github.com/IBM-Swift/Bridging">Bridging</a></h3>
-          <h4>Bridges funtions between macOS and Linux.</h4>
+          <h3><a href="https://github.com/IBM-Swift/CircuitBreaker">CircuitBreaker</a></h3>
```
Comment (ianpartridge) [inline packages.html]: Please leave CircuitBreaker in the Cloud section.

File: packages.html [hunk 3]
```diff
@@ -221,33 +195,61 @@ <h4>Sample application demonstrating Kitura and IBM Swift features.</h4>
...
           <h3><a href="https://github.com/IBM-Swift/TypeDecoder">TypeDecoder</a></h3>
           <h4>Allows inspection of native and complex types.</h4>
 
           <h3><a href="https://github.com/IBM-Swift/Configuration">Configuration</a></h3>
           <h4>Manages application configurations.</h4>
+
+          <h3><a href="https://github.com/IBM-Swift/FileKit">FileKit</a></h3>
+          <h4>Utility for resolving common paths.</h4>
+
+          <h3><a href="https://github.com/IBM-Swift/KituraContracts">Kitura Contracts</a></h3><a href="https://ibm-swift.github.io/KituraContracts/"><h5>API</h5></a>
```
Comment (ianpartridge) [inline packages.html]: Put this in Infrastructure.

File: packages.html (hunk 1 above)
Comment (helenmasters) [reply packages.html]: Done.

File: packages.html (hunk 2 above)
Comment (helenmasters) [reply packages.html]: Done.

File: packages.html (hunk 3 above)
Comment (helenmasters) [reply packages.html]: Done.

**Pull request merged**